bats/fb-verify-packages.bats

```diff
@@ -16,5 +16,10 @@ load foreman_helper
         exit 1
       fi
     fi
+    if [[ $FOREMAN_VERSION == 2.5* ]]; then
+      if rpm --query --all | grep --quiet --extended-regexp 'rh-ruby25'; then
```

Comment:
Why is `--extended-regexp` needed if it's not a regexp at all?

Reply:
Copy and paste from above since I know that works and does not hurt anything.